st2api/st2api/controllers/v1/pack_configs.py

```diff
@@ -63,3 +77,28 @@ def get_one(self, pack_ref):
         """
         # TODO: Make sure secret values are masked
         return self._get_one_by_pack_ref(pack_ref=pack_ref)
+
+    @request_user_has_permission(permission_type=PermissionType.PACK_CREATE)
+    @jsexpose(body_cls=ConfigUpdateRequestAPI, arg_types=[str])
+    def put(self, pack_uninstall_request, pack_ref):
+        """
+            Create a new config for the action.
+
+            Handles requests:
+                POST /configs/<pack_ref>
+        """
+
+        try:
+            config_api = ConfigAPI(pack=pack_ref, values=vars(pack_uninstall_request))
+            config_api.validate(validate_against_schema=True)
+        except jsonschema.ValidationError as e:
+            raise ValueValidationException(str(e))
+
+        config_content = yaml.safe_dump(config_api.values, default_flow_style=False)
+
+        configs_path = os.path.join(cfg.CONF.system.base_path, 'configs/')
+        config_path = os.path.join(configs_path, '%s.yaml' % config_api.pack)
+        with open(config_path, 'w') as f:
```

Comment:
Well, we would need to figure out permissions, but as far as HA setups go, it doesn't affect it in any way. Until we figure out the problem in general (as a part of StackStorm/discussions#204), it's up to a user to distribute configs and packs between the nodes.